### PR TITLE
Improve Figure.shift_origin docstring

### PR DESCRIPTION
**Pull request by michaelgrund**
**Description of proposed changes**

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Review by seisman — APPROVED** [review]

File: pygmt/figure.py [hunk 1]
```diff
@@ -395,8 +395,8 @@ def shift_origin(self, xshift=None, yshift=None):
         """
         Shift plot origin in x and/or y directions.
 
-        This method shifts plot origin relative to the current origin by
-        (*xshift*, *yshift*) and optionally append the length unit (**c**,
+        This method shifts the plot origin relative to the current origin
+        by (*xshift*, *yshift*). Optionally, append the length unit (**c**,
         **i**, or **p**).
```
Comment (weiji14) [inline pygmt/figure.py:400]: What's the default unit if c/i/p is not given?

Comment (seisman) [reply pygmt/figure.py:400]: The default is `c`.

**Comment by seisman** [comment]
I think it would be better if you can extend the docstrings to include more information according to the documentation at https://docs.generic-mapping-tools.org/latest/cookbook/options.html#plot-positioning-and-layout-the-x-y-options, especially **h** and **w** for previous plot's height and width.

File: pygmt/figure.py (hunk 1 above)
Comment (weiji14) [reply pygmt/figure.py:400]: Maybe add a sentence like the following to be clear?
```suggestion
        by (*xshift*, *yshift*). Optionally, append the length unit (**c**,
        **i**, or **p**). Default unit if not given is **c** for centimeter.
```

**Review by weiji14 — APPROVED** [review]
Ok, but can consider https://github.com/GenericMappingTools/pygmt/pull/1928#issuecomment-1134117224 in a separate PR upstream.

**Comment by michaelgrund** [comment]
> Ok, but can consider [#1928 (comment)](https://github.com/GenericMappingTools/pygmt/pull/1928#issuecomment-1134117224) in a separate PR upstream.

Agree!

**Pull request merged**